lib/propolis/Cargo.toml

```diff
@@ -33,6 +33,9 @@ once_cell.workspace = true
 crucible-client-types = { workspace = true, optional = true }
 crucible = { workspace = true, optional = true }
 oximeter = { workspace = true, optional = true }
+nexus-client.workspace = true
+internal-dns.workspace = true
+omicron-common.workspace = true
```

Comment:
These should be optional and gated on `crucible-full` like the other bits.

Reply:
Done.